gotham-theme.el

```diff
@@ -974,6 +974,17 @@ depending on DISPLAY for keys which are either :foreground or
    (rainbow-delimiters-depth-9-face :foreground red)
    (rainbow-delimiters-unmatched-face :foreground base2 :background base6)
 
+   ;; realgud
+   (realgud-overlay-arrow1         :foreground green)
+   (realgud-overlay-arrow2         :foreground yellow)
+   (realgud-overlay-arrow3         :foreground orange)
+   (realgud-bp-enabled-face        :inherit error)
+   (realgud-bp-disabled-face       :inherit secondary-selection)
+   (realgud-bp-line-enabled-face   :foreground red)
+   (realgud-bp-line-disabled-face  :inherit secondary-selection)
+   (realgud-line-number            :inherit sml/line-number)
```

Comment:
Please do not inherit from third-party packages.  Doing this will only use the inherited definition once the third-party package has been loaded.

Reply:
Ok. Fair enough. How is 39d07ab ?

Reply:
Looks good to me.